jupyterlab/handlers/extension_manager_handler.py

```diff
@@ -108,6 +132,7 @@ def list_extensions(self):
                 latest_version=pkg_info['latest_version'],
                 installed_version=data['version'],
                 status=status,
+                pkg_type='pre-built'
```

Comment:
Should this be `prebuilt`? (I think it was the spelling used in https://github.com/jupyterlab/jupyterlab/issues/9237) 

```suggestion
                pkg_type='prebuilt'
```

Reply:
No strong opinion, just to keep things consistent if we start using this term.